my-app/src/App.css

```diff
@@ -100,3 +100,65 @@ body {
 .card-body a:hover {
   background-color: #bb1919;
 }
+
+
+/* style for NewsContent component */
```

Comment:
Are you sure it's a necessary comment here?

Reply:
hmm it's just to know where am I